cranelift/codegen/src/binemit/stackmap.rs

```diff
@@ -15,7 +18,8 @@ const NUM_BITS: usize = core::mem::size_of::<Num>() * 8;
 /// The first value in the bitmap is of the lowest addressed slot on the stack.
 /// As all stacks in Isa's supported by Cranelift grow down, this means that
 /// first value is of the top of the stack and values proceed down the stack.
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, PartialEq, Eq)]
+#[cfg_attr(feature = "enable-serde", derive(Deserialize, Serialize))]
```

Comment:
To avoid the double-`#[cfg]` could this be `serde::Deserialize` and `serde::Serialize`?

Reply:
Nice call. Done.